notebooks/ldb-datacleaning.ipynb

```diff
@@ -0,0 +1,1782 @@
+{
```

Comment:
<p>In the raw csv file, downloaded direct of the Kaggle, the sep is equal to ',' and decimal is equal to '.'.</p><p> </p><p>I think you change the csv file.</p><p> </p><p>I suggest to use:</p><p><code>train=pd.read_csv('../data/raw/train.csv')</code></p><p> </p><p>Also, you are importing the matplotlib package but you don't use it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/neo-empresarial/house_pricing_challenge/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
I removed the matplotlib plackage and I also changed the the script for dowloaded the csv file